src/validators/StringValidator.ts

```diff
@@ -1,5 +1,18 @@
+import * as EmailValidator from 'email-validator';
+import { ConstraintError } from '..';
```

Comment:
Please remove all the `'..'` type imports and target the actual source files instead of the re-exports in `index.ts`